.changeset/loud-llamas-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@vercel/postgres': patch
+---
+
+Removed an inline condition to simplify the code just a tiny bit
```

Comment:
Not sure I'd even include a changeset for something like this which has no effect for users. 

Having the changeset means this will create a new release, and people will be bothered with upgrading.

Reply:
I didn't think there was a reason to either, but i got the message on github so i thought it was mandatory.  
I will remove it.